Google.Ads.GoogleAds/examples/CampaignManagement/ValidateAd.cs

```diff
@@ -122,13 +132,13 @@ public void Run(GoogleAdsClient client, long customerId, long adGroupId)
                     });
 
                 // Since validation is ON, result will be null.
-                Console.WriteLine("Expanded text ad validated successfully.");
+                Console.WriteLine("Responsive search ad validated successfully.");
```

Comment:
Since the idea in this example is for the request to trigger the catch block below, this print statement should never be executed, right?